Build problem for 1.5.2

**Describe the bug**
Fails to build.

**To Reproduce**
```
cargo +nightly  install-update koji
    Updating registry 'https://github.com/rust-lang/crates.io-index'

Package  Installed  Latest  Needs update
koji     v1.5.1     v1.5.2  Yes

Updating koji
```
...snip...
```
   Compiling cocogitto v5.2.0
   Compiling koji v1.5.2
error[E0061]: this function takes 7 arguments but 6 arguments were supplied
   --> /Users/berkus/.cargo/registry/src/github.com-1ecc6299db9ec823/koji-1.5.2/src/lib/commit.rs:43:15
    |
43  |     cocogitto.conventional_commit(&commit_type, scope, summary, body, None, is_breaking_change)?;
    |               ^^^^^^^^^^^^^^^^^^^-------------------------------------------------------------- an argument of type `bool` is missing
    |
note: associated function defined here
   --> /Users/berkus/.cargo/registry/src/github.com-1ecc6299db9ec823/cocogitto-5.2.0/src/lib.rs:364:12
    |
364 |     pub fn conventional_commit(
    |            ^^^^^^^^^^^^^^^^^^^
help: provide the argument
    |
43  |     cocogitto.conventional_commit(&commit_type, scope, summary, body, None, is_breaking_change, /* bool */)?;
    |                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Expected behavior**
Builds.

**Screenshots**
N/A

**Additional context**
N/A

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -15,17 +15,17 @@ name = "koji"
 path = "src/lib/lib.rs"
 
 [dependencies]
-anyhow = "1.0.61"
-clap = { version = "3.2.16", features = ["derive"] }
-cocogitto = { version = "5.1.0", default-features = false }
+anyhow = "1.0.65"
+clap = { version = "4.0.8", features = ["derive"] }
+cocogitto = { version = "5.2.0", default-features = false }
 conventional_commit_parser = "0.9.4"
 dirs = "4.0.0"
 emojis = "0.4.0"
 git2 = "0.15.0"
 indexmap = "1.9.1"
-openssl-sys = { version = "0.9.75", features = ["vendored"] }
-requestty = "0.4.1"
-serde = { version = "1.0.143", features = ["derive"] }
+openssl-sys = { version = "0.9.76", features = ["vendored"] }
+requestty = "0.5.0"
+serde = { version = "1.0.145", features = ["derive"] }
 toml = "0.5.9"
 
 [dev-dependencies]

--- a/src/lib/commit.rs
+++ b/src/lib/commit.rs
@@ -40,7 +40,15 @@ pub fn commit(
 ) -> Result<()> {
     let cocogitto = CocoGitto::get()?;
 
-    cocogitto.conventional_commit(&commit_type, scope, summary, body, None, is_breaking_change)?;
+    cocogitto.conventional_commit(
+        &commit_type,
+        scope,
+        summary,
+        body,
+        None,
+        is_breaking_change,
+        false,
+    )?;
 
     Ok(())
 }

